astropy/utils/diff.py

```diff
@@ -163,10 +163,11 @@ def where_not_allclose(a, b, rtol=1e-5, atol=1e-8):
     ----------
     a, b : array-like
         Input arrays to compare.
-
     rtol, atol : float
         Relative and absolute tolerances as accepted by
         :func:`numpy.allclose`.
+    return_maxdiff : bool
+        Return the maximum of absolute and relative differences.
 
     Returns
```

Comment:
Docstring should be updated to say it will return extra stuff given the new option?